src/components/Drawer.js

```diff
@@ -99,6 +93,7 @@ const Drawer = React.createClass({
   _animateComponent (transition, extraOptions) {
     const el = this._component;
     const options = Object.assign({
+      delay: 50,
```

Comment:
Valid question.  Not sure but I would guess yes.  We might have to do some testing to dial it in.  I can increase it to 100 ms if we feel a longer time would be better initially.